### PR TITLE
UIDATIMP-589 Match profile: Add Instance UUID as a match option,

### PR DESCRIPTION
**Pull request by IvanKryzhanovskyi**
## Purpose
To add support for matching by UUID for Inventory Instance, Holding and Item records

## Approach
1. Additonal option has been added to appropriate dropdowns
2. getFieldMatchedWithCategory method has been created and used for retrieving matched result with category

## Ticket
[UIDATIMP-589](https://issues.folio.org/browse/UIDATIMP-589)
[UIDATIMP-590](https://issues.folio.org/browse/UIDATIMP-590)
[UIDATIMP-591](https://issues.folio.org/browse/UIDATIMP-591)

## Screenshot
image

### CODE REVIEW

File: src/utils/getFieldsMatchedWithCategory.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { fieldsConfig } from './fields-config';
+import { getCategoryId } from './getDropdownOptions';
+
+export const getFieldMatchedWithCategory = (value, recordType, intl) => {
+  if (!value || !recordType) {
+    return undefined;
+  }
+
+  const isMarcRecord = recordType.toLowerCase().includes('marc');
```
Comment (mariia-aloshyna) [inline src/utils/getFieldsMatchedWithCategory.js:10]: Can `isMARCType` util be used here?

File: src/utils/getFieldsMatchedWithCategory.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+  if (isMarcRecord) {
+    return value;
+  }
+
+  const field = fieldsConfig.find(item => item.value === value);
+  const fieldValue = intl.formatMessage({ id: field.label });
+  const categoryValue = intl.formatMessage({ id: getCategoryId(field).label });
```
Comment (mariia-aloshyna) [inline src/utils/getFieldsMatchedWithCategory.js:18]: Consider moving `getCategoryId(field).label` to a variable

File: src/utils/getFieldsMatchedWithCategory.js (hunk 1 above)
Comment (IvanKryzhanovskyi) [reply src/utils/getFieldsMatchedWithCategory.js]: Done

File: src/utils/getFieldsMatchedWithCategory.js (hunk 2 above)
Comment (IvanKryzhanovskyi) [reply src/utils/getFieldsMatchedWithCategory.js]: Done

**Pull request merged**